default.nix

```diff
@@ -1,6 +1,12 @@
 { nurpkgs ? import <nixpkgs> {} # For nixpkgs dependencies used by NUR itself
   # Dependencies to call NUR repos with
-, pkgs ? null
+, pkgs ? import <nixpkgs>
```

Comment:
If we have this than https://github.com/nix-community/NUR/blob/05351320d5ece2eb43cbd991edd4d1f56f5ca17f/lib/evalRepo.nix#L13
becomes deadcode. So would probably should drop if if we want to allow this type of interface.